### PR TITLE
Hooli style popup

### PR DESCRIPTION
**Pull request by Lara-trost**
### task name

[Demo](https://lara-trost.github.io/hooli-style-popup/) |
[Code base](https://github.com/Lara-trost/hooli-style-popup.git)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/trostinetska/hooli-style-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    <meta name="description" content="popup">
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <title>Popup</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <ul class="crumbs">
+                <li class="crumb"><a href="#" tabindex="0">Mail</a></li>
```
Comment (A-Ostrovnyy) [inline submissions/trostinetska/hooli-style-popup/index.html:15]: By default, all interactive elements have a focus. no need to add `tabindex` attribute unless you want to change the default order.

File: submissions/trostinetska/hooli-style-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                <li class="crumb"><a href="#" tabindex="0">Images</a></li>
+                <li class="crumb">
+                <input type="checkbox" id="checkbox_toggle" name="checkbox_toggle" class="check_input" tabindex="0">
+                <label for="checkbox_toggle" class="check" >
+                    <img src="./icons/popup-button.png" alt="start-menu">
+                </label>
+                    <nav class="popup">
+                        <ul class="popup_list">
+                            <li class="popup_item"><a class="popup_link" href="#" tabindex="0"><img class="icon" src="./icons/calendar.png" alt="calendar"><p>Calendar</p></a></li>
```
Comment (A-Ostrovnyy) [inline submissions/trostinetska/hooli-style-popup/index.html:24]: [Read this article](https://htmlacademy.ru/blog/articles/alt-text) to find out how and when populate `alt` attribute

File: submissions/trostinetska/hooli-style-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                            <li class="popup_item"><a class="popup_link" href="#" tabindex="0"><img class="icon" src="./icons/mail.png" alt="mail"><p>Mail</p></a></li>
```
Comment (A-Ostrovnyy) [inline submissions/trostinetska/hooli-style-popup/index.html:25]: Use Prettier to prettify your code)

File: submissions/trostinetska/hooli-style-popup/index.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                                <li class="popup_item"><a class="popup_link" href="#" tabindex="0"><img class="icon" src="./icons/chat.png" alt="chat"><p>Chat</p></a></li>
+                                <li class="popup_item"><a class="popup_link" href="#" tabindex="0"><img class="icon" src="./icons/classroom.png" alt="classroom"><p>Classroom</p></a></li>
+                                <li class="popup_item"><a class="popup_link" href="#" tabindex="0"><img class="icon" src="./icons/collection.png" alt="collection"><p>Collection</p></a></li>
+                                <li class="popup_item"><a class="popup_link" href="#" tabindex="0"><img class="icon" src="./icons/mars.png" alt="mars"><p>Mars</p></a></li>
+                                <li class="popup_item"><a class="popup_link" href="#" tabindex="0"><img class="icon" src="./icons/blogs.png" alt="blogs"><p>Blogs</p></a></li>
+                                <li class="popup_item"><a class="popup_link" href="#" tabindex="0"><img class="icon" src="./icons/promo.png" alt="promo"><p>Promos</p></a></li>
+                            </ul>
+                            <p>Even more from Kottans</p>
```
Comment (A-Ostrovnyy) [inline submissions/trostinetska/hooli-style-popup/index.html:48]: Logically it's link, pleas use propper HTML tag)

File: submissions/trostinetska/hooli-style-popup/index.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+                            <li class="popup_item"><a class="popup_link" href="#" tabindex="0"><img class="icon" src="./icons/videos.png" alt="videos"><p>Video</p></a></li>
+                            <li class="popup_item"><a class="popup_link" href="#" tabindex="0"><img class="icon" src="./icons/search.png" alt="search"><p>Search</p></a></li>
+                            <li class="popup_item"><a class="popup_link" href="#" tabindex="0"><img class="icon" src="./icons/photos.png" alt="fotos"><p>Foto</p></a></li>
+                            <li class="popup_item"><a class="popup_link" href="#" tabindex="0"><img class="icon" src="./icons/apps.png" alt="apps"><p>Apps</p></a></li>
+                            <li class="popup_item"><a class="popup_link" href="#" tabindex="0"><img class="icon" src="./icons/documents.png" alt="documents"><p>Document</p></a></li>
+                        </ul>
+                        <input type="checkbox" id="popup_more" name="popup_more" class="popup_more" tabindex="0">
```
Comment (A-Ostrovnyy) [inline submissions/trostinetska/hooli-style-popup/index.html:37]: Add styles on focus state

File: submissions/trostinetska/hooli-style-popup/index.html [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <title>Popup</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <ul class="crumbs">
+                <li class="crumb"><a href="#" tabindex="0">Mail</a></li>
+                <li class="crumb"><a href="#" tabindex="0">Images</a></li>
```
Comment (A-Ostrovnyy) [inline submissions/trostinetska/hooli-style-popup/index.html:16]: Fix styles on hover: align content inside hovered border + avoid menu item jumps
image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

File: submissions/trostinetska/hooli-style-popup/index.html (hunk 1 above)
Comment (Lara-trost) [reply submissions/trostinetska/hooli-style-popup/index.html:15]: Thanks a lot! It was done.

File: submissions/trostinetska/hooli-style-popup/index.html (hunk 5 above)
Comment (Lara-trost) [reply submissions/trostinetska/hooli-style-popup/index.html:37]: I'm not sure I follow. Для яких елементів? Я можу додати focus для лінка. input стає невидимим, чи йому теж потрібен focus?

Comment (A-Ostrovnyy) [reply submissions/trostinetska/hooli-style-popup/index.html:37]: He needs focus too) you must indicate to user what interactive element in focus right now)

File: submissions/trostinetska/hooli-style-popup/index.html [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+                      src="./icons/calendar.png"
+                      alt=""
+                    />Calendar</a
+                  >
+                </li>
+                <li class="popup_item">
+                  <a class="popup_link" href="#"
+                    ><img class="icon" src="./icons/mail.png" alt="" />Mail</a
+                  >
+                </li>
```
Comment (A-Ostrovnyy) [inline submissions/trostinetska/hooli-style-popup/index.html:42]: Fix your styles on focus)
image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Awesome!
One last issue)

File: submissions/trostinetska/hooli-style-popup/index.html (hunk 7 above)
Comment (Lara-trost) [reply submissions/trostinetska/hooli-style-popup/index.html:42]: Thanks) I don't understand, what do I need to do? Should I delete focus-within and work only with focus? Or...?

Comment (A-Ostrovnyy) [reply submissions/trostinetska/hooli-style-popup/index.html:42]: By default browser use `outline` to draw an outline around the focused element, and you use border on focus. You must choose what approach you will use.

Comment (Lara-trost) [reply submissions/trostinetska/hooli-style-popup/index.html:42]: Thank you) I'll try to do this.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**